decidim-admin/app/queries/decidim/admin/newsletter_recipients.rb

```diff
@@ -41,6 +37,15 @@ def query
 
       private
 
+      def recipients_base_query
+        Decidim::User.available
+                     .where(organization: @form.current_organization)
+                     .where.not(newsletter_notifications_at: nil)
+                     .where.not(email: nil)
+                     .where.not(confirmed_at: nil)
+                     .not_deleted
```

Comment:
`.available` is implemented like: `where(deleted_at: nil, blocked: false, managed: false)`, so having the `not_deleted` added to the query is not needed. 

We also could use the `.confirmed` scope. 

```suggestion
        Decidim::User.available
                     .where(organization: @form.current_organization)
                     .where.not(newsletter_notifications_at: nil)
                     .where.not(email: nil)
                     .confirmed
```